### PR TITLE
Ryan dev3 20231121

### PR DESCRIPTION
**Pull request by ryanwooa**
1. adjust my part of scss files to new folder, and refactor part of them
2. refactor Todolist compoent to Layout folder
3. change name of "ListNav" to "AppNav"
4. add DUMMY_Data, Path: public/dummy-data.js

### CODE REVIEW

File: styles/layout/AppNav.module.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+@use "../scss/utils/variable" as *;
+
+.app_nav {
+  position: sticky;
+  top: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 23px;
```
Comment (zli213) [inline styles/layout/AppNav.module.scss:9]: Could this one be replaced with` $paddingsm: 1rem;`or `$paddingMd: 2rem;` ?

File: styles/layout/AppNav.module.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  height: $navHeight;
+  line-height: $navHeight;
+  z-index: 10;
+  background-color: $primaryColor;
+  color: rgb(255, 255, 255);
```
Comment (zli213) [inline styles/layout/AppNav.module.scss:14]: Could this one be replaced with:
$primaryColor: #de473a;
$secondaryColor: #fefdfc;
$accentColor: #ff7383;
$textPrimaryColor: #333333;
$textSecondaryColor: #555555; ?

File: styles/layout/Settings.module.scss [hunk 3]
```diff
@@ -13,7 +13,7 @@
   max-height: 1000px;
 }
 
-.setting-left {
+.setting_left {
   padding: 10px;
```
Comment (zli213) [inline styles/layout/Settings.module.scss:17]: Please replace px with rem, keep same style

File: styles/layout/Settings.module.scss [hunk 4]
```diff
@@ -26,3 +26,8 @@
     list-style: none;
   }
 }
+
+.grey_cover {
+  background-color: rgba($color: #000000ab, $alpha: 0.4);
```
Comment (zli213) [inline styles/layout/Settings.module.scss:31]: how about #000000ab ?

File: styles/layout/TodoList.module.scss [hunk 5]
```diff
@@ -1,10 +1,8 @@
 .list_editor {
   padding: 0px 55px 84px;
```
Comment (zli213) [inline styles/layout/TodoList.module.scss:2]: same point

File: components/layout/Settings.js [hunk 6]
```diff
@@ -1,14 +1,14 @@
-import "./Settings.scss";
+import styles from "../../styles/layout/Settings.module.scss"
```
Comment (zli213) [inline components/layout/Settings.js:1]: Use lowercase for the first letter of file name

**Review by zli213 — COMMENTED** [review]
Please check it

File: components/layout/Settings.js (hunk 6 above)
Comment (ryanwooa) [reply components/layout/Settings.js]: I adjust all above styles with variables. Please check.

Comment (zli213) [reply components/layout/Settings.js]: OK

**Pull request merged**